cadc-vos-server/src/main/java/org/opencadc/vospace/server/actions/UpdateNodeAction.java

```diff
@@ -179,6 +179,12 @@ public static Node updateProperties(Node serverNode, Node clientNode, NodePersis
                 scn.inheritPermissions = ccn.inheritPermissions;
             }
         }
+
+        // Remove the special Creator JWT property from the client node before copying into the server node as it will never be persisted.
```

Comment:
there are two comments above with `TODO` related to admin changing ownership. I would remove the first one in doAction and move this block of code up below the one near the start of this method as that will turn into the admin block if we decide to implement it.